### PR TITLE
Release notes 20.1.1

### PR DESCRIPTION
**Pull request by Amruta-Ranade**
Closes #7399

### CODE REVIEW

File: releases/v20.1.1.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+- It is now possible to drain a node without shutting down the process, using `cockroach node drain`. This makes it easier to integrate with service managers and orchestration: it now becomes safe to issue `cockroach node drain` and then separately stop the service via a process manager or orchestrator. Without this new mode, there is a risk to misconfigure the service manager to auto-restart the node after it shuts down via `quit`, in a way that's surprising or unwanted. The new command `node drain` also recognizes the new `--drain-wait` flag. [#47692][#47692]
+- The commands [`cockroach quit`](../v20.1/cockroach-quit.html) and `cockroach node drain` now reports a "work remaining" metric on their standard error stream. The value reduces until it reaches `0` to indicate that the graceful shutdown has completed server-side. An operator can now rely on `cockroach node drain` to obtain confidence of a graceful shutdown prior to terminating the server process. [#47692][#47692]
```
Comment (rmloveland) [inline releases/v20.1.1.md:71]: "commands now report" (singular)?

File: releases/v20.1.1.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+- `TRUNCATE` can now run on temporary tables, fixing a bug in v20.1 where temporary tables could not be truncated, resulting in an error `unexpected value: nil`. [#48078][#48078]
+- Fixed a bug in which `(tuple).*` was only expanded to the first column in the tuple and the remaining elements were dropped. [#48290][#48290]
+- Fixed case where `PARTITION BY` and `ORDER BY` columns in window specifications were losing qualifications when used inside views. [#47715][#47715]
+- CockroachDB will no longer display a severe `internal error` upon certain privilege check failures via `pg_catalog` built-in functions. [#48242][#48242]
+- Fixed a bug where a read operation in a transaction with a past savepoint rollback would give an internal error for exceeding the maximum count of results requested [#48165][#48165]
+- Fixed a bug that could lead to data corruption or data loss if a replica was both the source of a snapshot and was being concurrently removed from the range and certain specific conditions exist inside RocksDB. This scenario is rare, but possible. [#48321][#48321]
```
Comment (rmloveland) [inline releases/v20.1.1.md:94]: I would move this to the top of the list! :-)

File: releases/v20.1.1.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+- The distinction between delete jobs for columns and dependent jobs for deleting indices, views and sequences is now better defined. [#48259][#48259]
+- Fixed incorrect results that could occur when casting negative intervals or timestamps to type `decimal`. [#48345][#48345]
+- Fixed an error that occurred when statistics collection was explicitly requested on a column with type `array`. [#48343][#48343]
+- Fixed a nil pointer dereference in Pebble's block cache due to a rare "double free" of a block. [#48346][#48346]
+- Fixed Pebble to properly mark `sstables` for compaction which contain range tombstones. This matches the behavior when using RocksDB and ensures that space used for temporary storage is reclaimed quickly. [#48346][#48346]
+- Fixed a bug introduced in v20.1 that could cause multiple index GC jobs to be created for the same schema change in rare cases. [#47818][#47818]
+- Fixed a bug where CockroachDB could return an internal error when performing a query with `CASE`, `AND`, `OR` operators in some cases when it was executed via the vectorized engine. [#48072][#48072]
+- TLS 1.3 is now disabled by default to improve compatibility with Java. [#48365][#48365]
```
Comment (rmloveland) [inline releases/v20.1.1.md:102]: This may also want to be up higher for Java users to find.

File: releases/v20.1.1.md [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+- Fixed a rare bug where stats were not automatically generated for a new table. [#48027][#48027]
+- Fixed a panic that could occur when `SHOW RANGES` or `SHOW RANGE FOR ROW` was called with a virtual table. [#48347][#48347]
+- Made SRV resolution non-fatal for join list records to align with the standard and improve reliability of node startup. [#48349][#48349]
+- Fixed a rare bug causing a range to deadlock and all the writes to the respective range to timeout. [#48303][#48303]
+- Fixed a long-standing bug where HTTP requests would start to fail with error 503 "`transport: authentication handshake failed: io: read/write on closed pipe`" and never become possible again until the node is restarted. [#48456][#48456]
+- Fixed a bug where vectorized queries on composite datatypes could sometimes return invalid data. [#48463][#48463]
```
Comment (rmloveland) [inline releases/v20.1.1.md:108]: I would also move this up.  Anything about data being incorrect/invalid/corrupted/etc. is something I would want to see first and foremost as a user.

File: releases/v20.1.1.md [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+- Re-allowed `diagnostics.forced_sql_stat_reset.interval`, `diagnostics.sql_stat_reset.interval` and `external.graphite.interval` to set to their maximum values (24hr, 24hr and 15min respectively). [#48760][#48760]
+- Fixed a bug where CockroachDB could encounter an internal error when a query with `LEFT SEMI` or `LEFT ANTI` join was performed via the vectorized execution engine. This is likely to occur only with `vectorize=on` setting. [#48751][#48751]
+- Fixed a bug where running `cockroach dump` on tables with interleaved primary keys would erroneously include an extra `CREATE UNIQUE INDEX "primary" ... INTERLEAVE IN PARENT` statement in the dump output. This made it impossible to reimport dumped data without manual editing. [#48776][#48776]
+- Fixed a bug where running `cockroach dump` on a table with collated strings would omit the collation clause for the data insertion statements. [#48832][#48832]
+- Fixed a bug where the migration for ongoing schema change jobs would cause the node to panic with an "index out of bounds" error upon encountering a malformed table descriptor with no schema change mutation corresponding to the job to be migrated. [#48838][#48838]
```
Comment (rmloveland) [inline releases/v20.1.1.md:121]: Is this the schema change crash-loop bug that I saw folks talking about in Slack?  If it is, I think we should move it higher in the list and mention that the node will panic again if it is restarted.  But I'm not sure if this is that bug.

**Comment by Amruta-Ranade** [comment]
TFTR, @rmloveland !!

**Pull request merged**